src/sentry/api/endpoints/organization_metrics_tag_details.py

```diff
@@ -23,6 +23,7 @@ class OrganizationMetricsTagDetailsEndpoint(OrganizationEndpoint):
 
     def get(self, request: Request, organization, tag_name) -> Response:
         metric_names = request.GET.getlist("metric") or []
+        tag_value_prefix = request.GET.get("tagValuePrefix") or ""
```

Comment:
nit: since the url of this endpoint scopes the response to tags, i would name this param `prefix` or `valuePrefix`